diff -u ../boost_1_75_0-orig/boost/rational.hpp ./boost/rational.hpp
--- ../boost_1_75_0-orig/boost/rational.hpp	2020-12-03 06:02:19.000000000 +0100
+++ ./boost/rational.hpp	2022-01-27 16:02:27.993848905 +0100
@@ -302,6 +302,9 @@
        return *this;
     }
     template <class T>
+    #if defined(__clang__)
+    __attribute__((no_sanitize("undefined")))
+    #endif
     BOOST_CXX14_CONSTEXPR typename boost::enable_if_c<rational_detail::is_compatible_integer<T, IntType>::value, rational&>::type operator*= (const T& i)
     {
        // Avoid overflow and preserve normalization
@@ -311,6 +314,9 @@
        return *this;
     }
     template <class T>
+    #if defined(__clang__)
+    __attribute__((no_sanitize("undefined")))
+    #endif
     BOOST_CXX14_CONSTEXPR typename boost::enable_if_c<rational_detail::is_compatible_integer<T, IntType>::value, rational&>::type operator/= (const T& i)
     {
        // Avoid repeated construction
